Hmm, seems to be an error with the test:

```
Script:
--
: 'RUN: at line 1';   rm -rf "/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-iphonesimulator-i386/DebugInfo/Output/debug_prefix_map.swift.tmp" && mkdir -p "/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-iphonesimulator-i386/DebugInfo/Output/debug_prefix_map.swift.tmp"
: 'RUN: at line 2';   /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/bin/swiftc -frontend -target i386-apple-ios7.0  -module-cache-path '/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/swift-test-results/i386-apple-ios7.0/clang-module-cache' -sdk '/Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk' -swift-version 4   -emit-module-path /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-iphonesimulator-i386/DebugInfo/Output/debug_prefix_map.swift.tmp/Globals.swiftmodule /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/test/DebugInfo/Globals.swift
: 'RUN: at line 3';   env SDKROOT= '/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/bin/swiftc' -module-cache-path '/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/swift-test-results/i386-apple-ios7.0/clang-module-cache' -swift-version 4  -g -debug-prefix-map /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/test/DebugInfo=/var/empty -debug-prefix-map /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-iphonesimulator-i386/DebugInfo/Output/debug_prefix_map.swift.tmp=/var/empty /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/test/DebugInfo/debug_prefix_map.swift -I /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-iphonesimulator-i386/DebugInfo/Output/debug_prefix_map.swift.tmp -emit-ir -o - | '/usr/bin/python' '/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/utils/PathSanitizingFileCheck' --sanitize BUILD_DIR='/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64' --sanitize SOURCE_DIR='/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift' --use-filecheck '/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/bin/FileCheck' /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/test/DebugInfo/debug_prefix_map.swift
--
Exit Code: 2

Command Output (stderr):
--
/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/test/DebugInfo/debug_prefix_map.swift:5:8: error: module 'Globals' was created for incompatible target i386-apple-ios7.0: /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-iphonesimulator-i386/DebugInfo/Output/debug_prefix_map.swift.tmp/Globals.swiftmodule
import Globals
       ^
FileCheck error: '-' is empty.
FileCheck command line:  /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/bin/FileCheck /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/swift/test/DebugInfo/debug_prefix_map.swift

--
```

any idea how I can fix?

--- a/test/DebugInfo/debug_prefix_map.swift
+++ b/test/DebugInfo/debug_prefix_map.swift
@@ -1,6 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-swift-frontend -emit-module-path %t/Globals.swiftmodule %S/Globals.swift
-// RUN: %swiftc_driver -g -debug-prefix-map %/S=/var/empty -debug-prefix-map %t=/var/empty %/s -I %t -emit-ir -o - | %FileCheck %s
+// RUN: %target-swiftc_driver -g -debug-prefix-map %/S=/var/empty -debug-prefix-map %t=/var/empty %/s -I %t -emit-ir -o - | %FileCheck %s
 
 import Globals
 

